extensions/wikia/Forum/ForumExternalController.class.php

```diff
@@ -32,6 +28,14 @@ public function policies() {
 	 */
 
 	public function swapOrder() {
+		try {
+			// SUS-4042: Validate edit token
```

Comment:
Just a nitpick, but could we not have this snippet copied 4 times. Perhaps make a function that takes a function which will be invoked when the token is valid?